validator/validate.py

```diff
@@ -13,6 +14,7 @@
     "visual_selection": t.List[Position]
 }
 
+
```

Comment:
line no. 63
def is_valid_json_str(json_str: str) -> bool:
    return is_valid_content(json.loads(json_str))

Can't we directly call is_valid_content() ?

Reply:
If you're willing to parse json string, then yes. User will have both the functions and can choose accordingly.